### PR TITLE
NW6 | Rabia Avci | HTML-CSS | Get Forms Workshop | 

### PR DESCRIPTION
**Pull request by RbAvci**

### CODE REVIEW

File: get-forms/index.html [hunk 1]
```diff
@@ -3,11 +3,41 @@
...
     <meta name="description" content="My Github Search form" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
-    <form action="https://github.com/search" method="get"></form>
+    <div class="form">
+      <form action="https://github.com/search" method="get">
+        <div>
+          <label for="q">Search Field</label>
+          <input type="text" name="q" />
```
Comment (PERicci) [inline get-forms/index.html:15]: I suggest inserting "required" in this input, preventing the user does a blank search.

**Review by PERicci — CHANGES_REQUESTED** [review]
Just an adition in the search input.

File: get-forms/index.html (hunk 1 above)
Comment (RbAvci) [reply get-forms/index.html:15]: Thank you, Pedro done ✔️

**Comment by Ara225** [comment]
Good work :)

**Pull request closed without merging**